pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -1365,3 +1363,117 @@ def _init_data_default_frequencies(coldata, to_ts_types):
 def _start_stop_from_periods(periods):
     start, stop = _get_min_max_year_periods(periods)
     return start_stop(start, stop + 1)
+
+
+def get_profile_filename(station_or_region_name, obs_name, var_name_web):
+    return f"{station_or_region_name}_{obs_name}_{var_name_web}.json"
+
+
+def process_profile_data(
```

Comment:
Please add some comments about what that methos actually does

Reply:
Done